_posts/2015-09-4-automated-testing-and-continuous-integration.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Automated Testing with PHPUnit and Continuous Integration"
```

Comment:
It's fine go ahead use this title it's expressive, the current one is
wooden :)

Warm Regards,
Geshan Manandhar
On Sep 4, 2015 8:06 AM, "sachit-singh" <notifications@github.com> wrote:

> In _posts/2015-09-4-automated-testing-and-continuous-integration.md
> <https://github.com/younginnovations/blog.yipl.com.np/pull/43#discussion_r38720196>
> :
>
> > @@ -0,0 +1,25 @@
> > +---
> > +layout: post
> > +title: "Automated Testing with PHPUnit and Continuous Integration"
>
> i thought it would be too long...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/younginnovations/blog.yipl.com.np/pull/43/files#r38720196>
> .
>